source/_posts/Proof-of-concept-met-azure-batch.md

```diff
@@ -0,0 +1,447 @@
+---
+title: Proof of concept met Azure Batch
+date: 2019-05-01 19:54:37
+tags:
+---
+
+Case omschrijving 
+---
+
+Net als in mijn vorige blog post (https://cloudrepublic.github.io/2019/03/28/Proof-of-concept-met-durable-functions/) heb ik een Proof of Concept gemaakt om grote hoeveelheden XML bestanden te transformeren het gaat dan 3000 bestanden per keer met een totale grote van 18 gigabyte. Voor dit artikel kan ik wegens privacy niet de echte data gebruiken en heb ik een dataset gebruikt van https://www.kaggle.com/datasets het gaat hier om een dataset met landen en de wijnen welke uit het desbetreffende land komen. 
```

Comment:
[Proof of concept met durable functions](https://cloudrepublic.github.io/2019/03/28/Proof-of-concept-met-durable-functions)